src/Components/PaperEditor/ProgrammeScriptContainer/reset-word-timings.js

```diff
@@ -0,0 +1,106 @@
+// Updates index numbers of programme script elements
+const reindexList = (elements) => {
+  const reindexedList = elements.map((element, i) => {
+    element.index = i;
+
+    return element;
+  });
+
+  return reindexedList;
+};
+
+const updateWordTimings = (elements) => {
+  const reindexedList = reindexList(elements);
+  const updatedElementTimings = reindexedList.reduce((result, element) => {
+
+    // Re-calcultate word timings
+    if (element.type === 'paper-cut') {
+      const paperCut = element;
```

Comment:
probably mutates the original element 🤔 would need to think of a way to deep clone the element